docs/ADR-54-oracle-for-mana-pricing.md

```diff
@@ -0,0 +1,48 @@
+# Use an Oracle for MANA pricing according to USD rate
+
+## Statement of the problem
+
+Some features require the user to pay a certain amount of MANA for execution. These features include:
+
+- Publishing a collection, which charges the user a fixed amount of MANA depending on the rarities of the items being published with the collection
+
+- Claiming a name, where the user is charged a fixed amount of MANA to claim.
+
+- Purchasing Third Party item slots, which are charged according to the number of slots bought.
+
+Given the price fluctuations of the token, more importantly, the increase in price over time, these features have become more expensive for the user to consume, thus discouraging content creators from actually adding content to the platform, hurting Decentraland as a whole on the process.
```

Comment:
I will remove this paragraph and add  just something like

```The community has been used the DAO to define the price in MANA but pegged to the USD dollar```

Reply:
Done